Source/Common/ValuesJsonConverter.cs

```diff
@@ -128,28 +105,27 @@ public override void WriteJson(JsonWriter writer, object? value, JsonSerializer
                 throw new ArgumentNullException(nameof(value));
             }
 
-            if (serializer is null)
+            if (options is null)
```

Comment:
Should we be checking `value` for `null` here.

Reply:
Probably a bit lower down, yeah - we write a `null` if the `IValues` has a count of 0 but we don't actually check if its `null`.